src/routes/babbles/chat/[id]/[[thread_id]]/+page.svelte

```diff
@@ -126,6 +126,9 @@
 
 			const parentMessage = parentMap.get(message.parentId)
 			if (!parentMessage) {
+				// this is a workaround for the case when the parent is lost to
+				// keep the rest of the thread
+				roots.push(parent)
```

Comment:
Is this a temporary fix? If so, I suggest adding `//FIXME:` or `//TODO:`

Reply:
Well, for now it is a workaround for lost messages. 

However there is a broader issue that we only keep MAX_MESSAGES (currently set to 100) messages locally in the history. This is not a big issue with chats because they are linear, but with babbles it is possible that a parent message is lost because of this and the conversation would be orphaned without this workaround too.